### ISSUE TITLE
BeatLeader incompatible with Chroma & Noodle Extensions

### ISSUE THREAD

**Issue by kNoAPP** [BeatLeader/beatleader-qmod#10]
Using v0.4.1, when loading BeatSaber with just BeatLeader, the game loads just fine. When loading either Chroma or Noodle Extensions or both with BeatLeader, game immediately crashes on start.

I absolutely love BeatLeader and the scoreboards & replay capabilities. The mod is quite popular among my Sabering group to keep our sessions competitive. I realize supporting other mods may be out of scope of this project, but I'd like to at least make mention of this particular compatibility combination. Chroma & Noodle Extensions enable so many additional new & creative songs.

This is more of a feature request than a bug report. So if its out of scope, feel free to just close this issue.

**Comment by NSGolova** [comment]
Hi, please send the crash logs(tombstones). Instructions: https://twitter.com/beatleader_/status/1549771916931747843

The last Quest release should be already compatible with Chroma/Noodles. At least the first versions I tested

**Comment by kNoAPP** [comment]
[tombstone_02.zip](https://github.com/BeatLeader/beatleader-qmod/files/9388101/tombstone_02.zip)

**Comment by NSGolova** [comment]
Try updating the Chroma/Noodle to the latest version. This crash should be fixed

**Comment by yoziming** [comment]
[tombstone_00.zip](https://github.com/BeatLeader/beatleader-qmod/files/9482015/tombstone_00.zip)

Same issue with all latest version.
NoodleExtensions Version 1.2.4-beta1
Chroma Version 2.6.4
BeatLeader Version 0.4.1

**Comment by Tadiian** [comment]
Yeah same here. Took me a while to figure out which mods did cause the crash. Figured it out in the end that it was BeatLeader combined with either Chroma and/or Noodles. All latest versions for 1.24.
